stores/menuStore.ts

```diff
@@ -0,0 +1,22 @@
+import { defineStore } from "pinia"
+import { ref, watch } from 'vue'
+
+export const useMenuStore = defineStore('menuStore', () => {
+    const menuOpen = ref(false)
+    const menuItems = {
+        "home": { "displayText": "topNav.home", "route": "/" },
+        "about": { "displayText": "topNav.about", "route": "/about" },
+        "submit": { "displayText": "topNav.submit", "route": "/submit" },
+    }
+
+    function toggleMenu() {
+        console.log("menu= ", menuOpen.value)
```

Comment:
should remove this log. we can check vue plugins to track state. 
for future reference, we should only keep logs for data fetching/submission and warnings/errors